### ISSUE TITLE
native injection capability

### ISSUE THREAD

**Issue by mgreen27** [Velocidex/velociraptor#1219]
We would like an injection detection capability.

Field ideas:
![image](https://user-images.githubusercontent.com/13081800/130932165-8ab328a5-99e2-45b7-b1a6-2497f3c759b0.png)

**Comment by scudette** [comment]
Can you please elaborate on this one? What exactly are we looking for?

If you just need page read/write/execution you can get that with the vad plugin.

**Comment by mgreen27** [comment]
Yes - the idea is to remove the need to run 3rd party scripts/tools for injection detection/leads.

**Comment by scudette** [comment]
We need more details - what scripts specifically? what plugins do we need?
